### PR TITLE
Fix Travis build

### PR DESCRIPTION
**Pull request by aitbw**
This PR aims to get a green Travis build by introducing the following changes:

* Fix unrecognized image format error when rendering invoice logo (see commit's description for details)
* Lock SQLite3 to version 1.3 ( (see commit's description for details)

### CODE REVIEW

File: app/models/spree/print_invoice_configuration.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@ module Spree
   class PrintInvoiceConfiguration < Preferences::Configuration
 
     preference :print_invoice_next_number, :integer, :default => nil
-    preference :print_invoice_logo_path, :string, :default => Spree::Config[:admin_interface_logo]
+    preference :print_invoice_logo_path, :string, :default => Spree::Config[:admin_interface_logo].end_with?('.svg') ? 'logo/solidus_logo.png' : Spree::Config[:admin_interface_logo]
```
Comment (mdesantis) [inline app/models/spree/print_invoice_configuration.rb]: What about make the line shorter?

Comment (aitbw) [reply app/models/spree/print_invoice_configuration.rb]: Fixed, @mdesantis! WDYT?

Comment (mdesantis) [reply app/models/spree/print_invoice_configuration.rb]: Much better!

Comment (mdesantis) [reply app/models/spree/print_invoice_configuration.rb]: But I'd use `if else end`, ternary operator don't look good when splitted on more lines; I'd do something like this:

```ruby
preference :print_invoice_logo_path, :string,
           :default => if Spree::Config[:admin_interface_logo].end_with?('.svg')
                         'logo/solidus_logo.png'
                       else 
                         Spree::Config[:admin_interface_logo]
                       end
```

File: app/models/spree/print_invoice_configuration.rb [hunk 2]
```diff
@@ -1,8 +1,11 @@
 module Spree
   class PrintInvoiceConfiguration < Preferences::Configuration
-
     preference :print_invoice_next_number, :integer, :default => nil
-    preference :print_invoice_logo_path, :string, :default => Spree::Config[:admin_interface_logo]
+
```
Comment (jacobherrington) [inline app/models/spree/print_invoice_configuration.rb]: Would you mind leaving a comment to explain why we are doing this?

Comment (aitbw) [reply app/models/spree/print_invoice_configuration.rb]: Sure thing!

Comment (aitbw) [reply app/models/spree/print_invoice_configuration.rb]: Done @jacobherrington! WDYT?

**Pull request merged**